src/Commands/IslandoraCommands.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+namespace Drupal\islandora\Commands;
+
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drush\Commands\DrushCommands;
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+/**
+ * A Drush commandfile.
+ *
+ * In addition to this file, you need a drush.services.yml
+ * in root of your module, and a composer.json file that provides the name
+ * of the services file to use.
+ *
+ * See these files for an example of injecting Drupal services:
+ *   - http://cgit.drupalcode.org/devel/tree/src/Commands/DevelCommands.php
+ *   - http://cgit.drupalcode.org/devel/tree/drush.services.yml
+ */
+class IslandoraCommands extends DrushCommands {
+
+  /**
+   * Module handler.
+   *
+   * @var Drupal\Core\Extension\ModuleHandlerInterface
+   */
+  protected $moduleHandler;
+
+  /**
+   * Constructor.
+   */
+  public function __construct(ModuleHandlerInterface $module_handler) {
+    $this->moduleHandler = $module_handler;
+  }
+
+  /**
+   * Install Solution Pack objects.
+   *
+   * @option module
+   *   The module for which to install the required objects.
+   * @option force
+   *   Force reinstallation of the objects.
+   * @usage drush -u 1 ispiro --module=islandora
+   *   Install missing solution pack objects for the "islandora" module.
+   * @usage drush -u 1 ispiro --module=islandora --force
```

Comment:
This style of user setting won't keep working. It can be specified as `-u1` or moved after the command. Most examples will probably need updating.